include/picongpu/plugins/binning/axis/LogAxis.hpp

```diff
@@ -0,0 +1,287 @@
+                    auto logMax = std::log2(toPICUnits(axisSplit.m_range.max, units));
+
+                    // do scaling calc here, on host and save it
+                    auto scaling = static_cast<decltype(logMax)>(axisSplit.nBins) / (logMax - logMin);
+
+                    auto nBins = axisSplit.nBins;
+                    if(axisSplit.enableOverflowBins)
+                    {
+                        nBins += 2;
+                    }
+
+                    lAK.initAxisSplit(logMin, logMax, nBins, scaling);
+                }
+
+
+                constexpr uint32_t getNBins() const
+                {
+                    return lAK.nBins;
+                }
+
+                double getUnitConversion() const
+                {
+                    return get_conversion_factor(units);
+                }
+
+
+                LogAxisKernel getAxisKernel() const
+                {
+                    return lAK;
+                }
+
+                BinWidthKernel getBinWidthKernel()
+                {
+                    // reset whenever binWidths changez
+                    static bool set = false;
+                    static pmacc::HostDeviceBuffer<T, 1> binWidthBuffer{axisSplit.nBins};
+                    if(!set)
+                    {
+                        auto db = binWidthBuffer.getHostBuffer().getDataBox();
+                        for(size_t i = 0; i < binWidths.size(); i++)
+                        {
+                            db[i] = binWidths[i];
+                        }
+                        binWidthBuffer.hostToDevice();
+                        set = true;
+                    }
+                    return BinWidthKernel(binWidthBuffer);
+                }
+
+                /**
+                 * @return bin edges in SI units
+                 */
+                std::vector<double> getBinEdgesSI()
+                {
+                    std::vector<double> binEdges;
+                    binEdges.reserve(axisSplit.nBins + 1);
+                    T factor = std::pow(2., (lAK.logMax - lAK.logMin) / axisSplit.nBins);
+                    T edge = axisSplit.m_range.min;
+                    for(size_t i = 0; i <= axisSplit.nBins; i++)
+                    {
+                        binEdges.emplace_back(toSIUnits(edge, units));
+                        edge = edge * factor;
+                    }
+                    return binEdges;
+                }
+            };
+
+
+            /**
+             * @details Creates a log Axis
+             * @tparam T_Attribute Type of the deposition functor (This is also the type of min, max and return type of
+             * the attrFunctor and if these types dont match this will throw an error)
+             * @param axisSplitting
+             * @param functorDescription
+             */
+            template<typename T_Attribute, typename T_FunctorDescription>
+            HINLINE auto createLog(AxisSplitting<T_Attribute> axSplit, T_FunctorDescription functorDesc)
```

Comment:
The thought process behind the naming was that this function will be called with the leading namespace, like ``axis::createLog(...)``, so it would be reasonably clear. 
But I am open to your suggestion if you still feel it is best to append Axis to the funcion names